kernels/volk/volk_32f_index_max_16u.h

```diff
@@ -109,7 +109,7 @@ volk_32f_index_max_16u_a_avx(uint16_t* target, const float* src0,
     currentValues  = _mm256_load_ps(inputPtr); inputPtr += 8;
     currentIndexes = _mm256_add_ps(currentIndexes, indexIncrementValues);
 
-    compareResults = _mm256_cmp_ps(maxValues, currentValues,14);
+    compareResults = _mm256_cmp_ps(currentValues, maxValues, _CMP_LE_OS);
```

Comment:
A couple queries here, noting that the `14` from the original code corresponds to `_CMP_GT_OS`:

(1) I believe that using `LE` (or `GE` if the first 2 arguments are swapped) just changes the index returned of the max value to being the -final- index versus the -first- index if using `LT` (or `GT`). Yes? We should pick which one we want to go with and document it in the header.

(2) Do you know the difference between the `_CMP_*_OS` (ordered, signaling) and `_CMP_*_OQ` (ordered, non-signaling)? I'm not sure it makes a difference here, but maybe it does?